src/main/java/com/vso/model/service/login/LoginSession.java

```diff
@@ -7,7 +7,7 @@
 public class LoginSession {
 
     private User loggedUser;
-    private int loggedId;
+    private long loggedId;
```

Comment:
who made this changes?? If it is not your code, please ask your team  to help you to decide if here should be int or long

Reply:
I checked in dev branch: here it should be int